README.md

```diff
@@ -104,6 +104,82 @@ let magicGrid = new MagicGrid({
 magicGrid.listen();
 ```
 
+#### Resizable Containers:
+
+**CSS Properties:**
+
+The following CSS properties are applied to the `.container` class to allow resizing and ensure proper overflow handling:
+
+```css
+.container {
+    width: 80%;
+    resize: both; /* Allows the container to be resizable both horizontally and vertically */
+    overflow: auto; /* Ensures that overflowing content will be scrollable */
+    border: 2px solid #ccc;
+    padding: 10px;
+    margin: 20px auto;
```

Comment:
We shouldn't set any of these properties. Any opinionated styling should be left to the user the library needs to be as generic as possible. Are you trying to explain the conditions where container resizing would be useful?